### PR TITLE
Prac 5 code review request

### PR DESCRIPTION
**Pull request by tananhnguyen**
@CynthiaJCU
@ArthurMy
@AdityaVijay1
@Abert-Alvaro

### CODE REVIEW

**Comment by ArthurMY** [comment]
@Nguyentananh98

**Comment by tananhnguyen** [comment]
@ArthurMY

**Comment by ArthurMY** [comment]
nice working

File: Practical 5/wimbledon.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+CP1404 Programming 2 - Practical 5
+NGUYEN TAN ANH - JC950881
+"""
+
+FILENAME = 'wimbledon.csv'
+
+
+def main():
+    """display the count of how many time champions and the countries that won the wimbledon"""
+    data_list = load_data(FILENAME)
```
Comment (ArthurMY) [inline Practical 5/wimbledon.py:11]: read the file out of the program

**Pull request merged**